README.md

```diff
@@ -25,7 +25,11 @@ settings] you may want:
             $('#redactor').redactor({
                 plugins: ['uploadcare'],
                 uploadcare: {
-                    // see https://uploadcare.com/documentation/widget/#configuration
+                    // styling options
+                    buttonLabel: 'Image / file'
+                    buttonBefore: 'video',
```

Comment:
Ah, what this does it that you can position a button before another button. Right now the button will always be placed on the last position in the toolbar ([docs](https://imperavi.com/redactor/docs/api/button/#api-addBefore)).
Yeah, video is an example in this case. I'm not sure how I can make that clear though.